apps/backend/src/config/mikro-orm.ts

```diff
@@ -0,0 +1,19 @@
+import { Options } from '@mikro-orm/core';
+
+const postGresHost = process.env.POSTGRES_HOST || 'localhost';
+const postGresPort = parseInt(process.env.POSTGRES_PORT) || 5432;
+const postGresUser = process.env.POSTGRES_USER || 'postgres';
+const postGresPassword = process.env.POSTGRES_PASSWORD || 'postgres';
+const postGresDatabase = process.env.POSTGRES_DATABASE || 'postgres';
+
+const config: Options = {
+  type: 'postgresql',
+  debug: true,
```

Comment:
Debug option could be dependent of  `process.env.NODE_ENV` variable :
```suggestion
  debug: process.env.NODE_ENV !== 'production',
```